packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -479,9 +499,11 @@ class WebAssetServer implements AssetReader {
     // browsers will refuse to render images/show video etc. If the tool
     // cannot determine a mime type, fall back to application/octet-stream.
     final String mimeType = mime.lookupMimeType(
-        file.path,
-        headerBytes: await file.openRead(0, mime.defaultMagicNumbersMaxLength).first,
-    ) ?? _kDefaultMimeType;
+          file.path,
+          headerBytes:
+              await file.openRead(0, mime.defaultMagicNumbersMaxLength).first,
+        ) ??
+        _kDefaultMimeType;
```

Comment:
Do I smell traces of `dart format .` here and there? :P

Reply:
Aww shoot - what a headache! Reverted